opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/CommonGivenStages.java

```diff
@@ -68,4 +75,17 @@ public SELF user_registered_in_opba_with_credentials(String username, String pas
 
         return self();
     }
+
+    public SELF ignore_validation_rules_table_contains_field_psu_id() {
+        IgnoreBankValidationRule bankValidationRule = IgnoreBankValidationRule.builder()
+                .protocol(BankProtocol.builder().id(1L).build())
+                .endpointClassCanonicalName("de.adorsys.opba.protocol.xs2a.service.xs2a.ais.AccountListingService")
```

Comment:
`.endpointClassCanonicalName(AccountListingService.class.getCanonicalName())` is way safer